system/settings/settings.xml

```diff
@@ -1620,6 +1620,18 @@
           <control type="toggle" />
         </setting>
       </group>
+      <group id="3">
+      <requirement>HAS_ZEROCONF</requirement>
```

Comment:
I'm unsure if we should keep this requirement or make it available globally.
@peak3d you said in slack, that you could make use of such a uuid as well?

Reply:
I can adjust it as seen necessary, it probably then would not be anymore inside `service` namespace but somewhere else if that's the case?